ui/src/components/PageUserInfo.tsx

```diff
@@ -62,7 +62,7 @@ const LoggedInUserInfoPanel: React.FC<UserInfo> = ({avatar, username}) => {
 
 const LoggedOutUserInfoPanel: React.FC = () => (
   <>
-    <a href={`${import.meta.env.VITE_API_URL}/oauth2/authorization/discord`} className="inline-block text-white text-3xl mt-8 mb-2 w-auto hover:underline">Log In{"\n"}With Discord</a>
+    <a href={`${import.meta.env.API_URL}/oauth2/authorization/discord`} className="inline-block text-white text-3xl mt-8 mb-2 w-auto hover:underline">Log In{"\n"}With Discord</a>
```

Comment:
For security reasons, Vite won't include environment variables in the front-end build unless it's prefixed with `VITE_`. (other tools have similar restrictions)

Reply:
Whoops - I'll revert that and have two fields 👍